demos/common/python/openvino/model_zoo/model_api/models/model.py

```diff
@@ -17,6 +17,12 @@
 import logging as log
 
 
+class WrapperError(RuntimeError):
```

Comment:
In each wrapper, the `WrapperError` is imported and called with `__model__` attribute for raising an error. Since all wrappers have `__model__`, you can create the common function for `Model` class like this to avoid the repetition:
```
def raise_error(self, message):
    raise WrapperError(self.__model__, message)
```